test/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests/OpenApiExampleFactoryTests.cs

```diff
@@ -0,0 +1,245 @@
+using System;
+using System.Text;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.OpenApi.Any;
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests
+{
+    [TestClass]
+    public class OpenApiExampleFactoryTests
+    {
+        [TestMethod]
```

Comment:
Could you use `DataTestMethod` instead of `TestMethod`, to inject test variables from outside?